utils/xbfmg.c

```diff
@@ -26,13 +26,11 @@
 
 void prtusage( char * );
 
-#define MXFLEN 125 /* Maximum length of bufrfile [path/]name. */
+#define MXFLEN 125 /**< Maximum length of bufrfile [path/]name. */
 
 /**
  * This function prints program usage information to standard output.
  *   
- * @param prgnam - [path/]name of program executable.
- *
  * @author J. Ator @date 2018-03-01
```

Comment:
@edwardhartnett Why was the `@param prgnam - [path/]name of program executable.` comment removed here?  Shouldn't the call parameter for the function be defined here?

Reply:
Doxygen is giving me a problem with this function that I am still exploring. Somehow it is being documented twice. I will circle around to this again when more of the warnings have been cleared up.